src/extensions/exceptions/exception-autocapture.ts

```diff
@@ -109,12 +109,16 @@ export class ExceptionObserver {
     }
 
     captureException(args: ErrorEventArgs, properties?: Properties) {
-        const errorProperties = errorToProperties(args)
-        const propertiesToSend = { ...properties, ...errorProperties }
+        if (this.isEnabled()) {
+            const errorProperties = errorToProperties(args)
+            const propertiesToSend = { ...properties, ...errorProperties }
 
-        const posthogHost = this.instance.config.ui_host || this.instance.config.api_host
-        errorProperties.$exception_personURL = posthogHost + '/person/' + this.instance.get_distinct_id()
+            const posthogHost = this.instance.config.ui_host || this.instance.config.api_host
+            errorProperties.$exception_personURL = posthogHost + '/person/' + this.instance.get_distinct_id()
 
-        this.sendExceptionEvent(propertiesToSend)
+            this.sendExceptionEvent(propertiesToSend)
+        } else {
+            console.warn('PostHog exception capture needs to be enabled first.')
+        }
```

Comment:
As mentioned in person I think the setting should only handle whether we automatically collect exceptions. It shouldn't bar them entirely.